From ce564f13431c6a57c2def4bdcd2991f2a90918cf Mon Sep 17 00:00:00 2001
From: "haotian.qu" <haotian.qu@amlogic.com>
Date: Fri, 7 Jul 2023 13:50:42 +0800
Subject: [PATCH 03/10] g12b: g12b support OTA feature. [1/3]

PD#SWPL-130114

Problem:
g12b support OTA feature.

Solution:
g12b support OTA feature.

Verify:
local

Signed-off-by: haotian.qu <haotian.qu@amlogic.com>
Change-Id: I153d5594e4d3e64e16739bcfb0775054c29cb249
---
 conf/machine/mesong12b-w400-weston-5.15.config | 4 ++--
 recipes-kernel/linux/5.15/g12b.cfg             | 1 -
 2 files changed, 2 insertions(+), 3 deletions(-)

diff --git a/conf/machine/mesong12b-w400-weston-5.15.config b/conf/machine/mesong12b-w400-weston-5.15.config
index 9d2f4323..840b9b65 100644
--- a/conf/machine/mesong12b-w400-weston-5.15.config
+++ b/conf/machine/mesong12b-w400-weston-5.15.config
@@ -68,8 +68,8 @@ DISTRO_FEATURES_append = " aml-libvphevcodec"
 DISTRO_FEATURES_append = " arm-isp "
 DISTRO_FEATURES_append = " softap"
 DISTRO_FEATURES_append = " aml-libgdc"
-#DISTRO_FEATURES_append = " swupdate"
-#DISTRO_FEATURES_append = " recovery"
+DISTRO_FEATURES_append = " swupdate"
+DISTRO_FEATURES_append = " recovery"
 
 DISTRO_FEATURES_append = " gst-aml-imgproc"
 DISTRO_FEATURES_append = " gst-aml-gfx2d"
diff --git a/recipes-kernel/linux/5.15/g12b.cfg b/recipes-kernel/linux/5.15/g12b.cfg
index eb3b85f1..1ed8c39b 100644
--- a/recipes-kernel/linux/5.15/g12b.cfg
+++ b/recipes-kernel/linux/5.15/g12b.cfg
@@ -47,7 +47,6 @@ CONFIG_F2FS_FS=n
 CONFIG_QUOTA=n
 CONFIG_VIRTUALIZATION=n
 CONFIG_IPV6=m
-CONFIG_FB=n
 CONFIG_CAN=n
 CONFIG_USB_NET_DRIVERS=n
 CONFIG_SERIAL_8250=n
-- 
2.25.1

